apps/events/views.py

```diff
@@ -1,10 +1,29 @@
 from django.http import HttpResponse
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import permission_required
+from .models import Event
+from django.contrib import messages
 
 
 def index(request):
     return HttpResponse('Hello World!')
 
 
+@permission_required('events.add_event', raise_exception=True)
```

Comment:
Can you also add the other decorators to this, like the ones used in students views?